try {
    throw "こんにちは"
} catch (error) {
    console.log("throw されたオブジェクト:", error)
}